Code/Framework/AzFramework/AzFramework/Physics/RagdollEMotionBus.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
AzFramework is a low level layer which is designed to deal with animation and physics in general, rather than specific implementation gems (e.g. EMotionFX, PhysX gems) so it shouldn't know about any particular gem.

Shape configurations, joint configurations etc are used by ragdolls and in other situations, so I don't think it's a good idea to have the naming specifically refer to ragdolls in all the events here.

I'd suggest changing the name of this bus to something like PhysicsConfigurationNotificationBus, and changing the events to something more generic e.g. OnRagdollNodeConfigurationChanged, OnJointConfigurationChanged, OnColliderConfigurationChanged, OnShapeConfigurationChanged.